### PR TITLE
adding docstring

### LINKED ISSUE #175: check if all functions have docstring

### PR DESCRIPTION
**Pull request by cedricserugendo**
add docstring on functions

### CODE REVIEW

File: api/app/endpoints/wms.py [hunk 1]
```diff
@@ -65,14 +83,15 @@ def parse_size(params) -> Size:
...
     if (height * width) > current_app.config["WMS"]["MAX_SIZE"]:
-        abort(400, "Total size is bigger than the maximaml allowed size")
+        abort(400, "Total size is bigger than the maximal allowed size")
     return Size(width=width, height=height)
 
 
 Position = namedtuple("Position", ("x", "y"))
 
 
 def parse_position(params) -> Size:
```
Comment (melhk) [inline api/app/endpoints/wms.py:93]: The type of the variable returned is not matching the type the function expects??

Comment (melhk) [inline api/app/endpoints/wms.py]: Why does this function return an other type than the one expected?

**Review by melhk — COMMENTED** [review]
1 comment to check

File: api/app/endpoints/wms.py (hunk 1 above)
Comment (cedricserugendo) [reply api/app/endpoints/wms.py]: This is a mistake, the function should return the namedtuple `Position` (see lines 293-296)
I'll fix it

**Review by melhk — APPROVED** [review]

**Pull request merged**